docs/configuration.md

```diff
@@ -629,7 +638,7 @@ editing the `conf` file in a text editor. Use the examples as reference.
     </tr>
 </table>
 
-### [native_pen_touch](https://localhost:47990/config/#native_pen_touch)
+### keybindings
```

Comment:
This option isn't available in the UI, but a "Open" button will still be generated.

We should probably add this to the UI instead of excluding the open button from being created just for this case.